docs/source/doc.gptable.rst

```diff
@@ -135,6 +135,179 @@ This ``additional_formatting`` parameter is best demonstrated by example:
          }
    ]
 
+Formatting methods
```

Comment:
Can we make it clearer that this relates to the additional formatting? Maybe it could be a subsection of that section and make reference to it in the description paragraph